src/index.html

```diff
@@ -17,16 +17,34 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form class="form">
+      <label
+        for="big-bar"
+        class="form__label form__label-big"
+        aria-label="Search big"
+      ></label>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <input
+        type="text"
+        data-qa="big"
+        placeholder="Try “Los Angeles“"
+        class="form__bar form__bar--big"
+        id="big-bar"
+      />
+
+      <label
+        for="small-bar"
+        class="form__label form__label-small"
+        aria-label="Search small"
+      ></label>
+
+      <input
+        type="text"
+        data-qa="small"
+        placeholder="Try “Los Angeles“"
+        class="form__bar form__bar--small"
+        id="small-bar"
+      />
```

Comment:
There should be empty lines between multiline sibling blocks of HTML to improve readability. Please add an empty line between the first <label> with its <input> and the second <label> with its <input>.